paper.tex

```diff
@@ -313,7 +313,28 @@ \subsubsection{Trading}
     \label{fig:trading_product}
 \end{figure}
 
-Next, session key $k$ should be exchanged between the data provider and consumers as shown in Figure \ref{fig:key_exchange}. The data provider can obtain public keys of each consumer from the registration contract. For each consumer, the data provider encrypts the session key and broker's signature with the consumer's public key and sends the ciphertext $Encrypt(k + Sign(k))$ to the product contract. Consumers listen to the smart contract event which is triggered when the ciphertext is updated, and decrypt the ciphertext to obtain the session key $k$ and signature $Sign(k)$.
+Next, session key $k$ should be exchanged between the data provider and consumers. To exchange data, some researchs\cite{Medrec} seek an off-chain solution which uses an end-to-end communication that is efficient and low-cost. However, it is hard to avoid that the data source is unavailable or the sender sends incorrect data intentionally. Instead of using off-chain solution, we use smart contract to transfer the session key\cite{3tierDataMarket} that not only ensures the consistency of the session key, the availability of the source and the traceability of the record, but also prevents malicious participants to cheat others.
```

Comment:
Typo, `some researches\cite{Medrec}....`.
Also since only one paper is cited here, I think change `some researches` to the authors' name here would be better.